client/network/src/service.rs

```diff
@@ -552,6 +553,30 @@ impl<B: BlockT + 'static, H: ExHashT> NetworkWorker<B, H> {
 	pub fn add_reserved_peer(&self, peer: String) -> Result<(), String> {
 		self.service.add_reserved_peer(peer)
 	}
+
+	/// Get a mutable reference to the Bitswap behaviour.
+	pub fn bitswap(&mut self) -> &mut libp2p_bitswap::Bitswap {
+		&mut self.network_service.bitswap
+	}
```

Comment:
I'm not a fan of exposing private types such as `libp2p_bitswap::Bitswap`. For the sake of any future maintainer, please duplicate the function definitions of bitswap.